cmd/explaintest/r/access_path_selection.result

```diff
@@ -35,7 +35,7 @@ StreamAgg	1.00	root		funcs:max(test.access_path_selection._tidb_rowid)->Column#4
 └─Limit	1.00	root		offset:0, count:1
   └─TableReader	1.00	root		data:Limit
     └─Limit	1.00	cop[tikv]		offset:0, count:1
-      └─TableFullScan	1.25	cop[tikv]	table:access_path_selection	keep order:true, desc, stats:pseudo
```

Comment:
why is it changed ?

Reply:
`_tidb_rowid` column is set to be not null, which effects the cost estimation.